update_handlers.go

```diff
@@ -0,0 +1,131 @@
+package handlers
+
+import (
+    "os"
+    "time"
+    "github.com/gofiber/fiber/v2"
+    "github.com/golang-jwt/jwt/v4"
+    "github.com/sandbox-science/online-learning-platform/configs/database"
+    "github.com/sandbox-science/online-learning-platform/internal/entity"
+    "github.com/sandbox-science/online-learning-platform/internal/utils"
+)
+
+// UpdateUsername updates a user's username
+func UpdateUsername(c *fiber.Ctx) error {
+    var data map[string]string
+
+    if err := c.BodyParser(&data); err != nil {
+        return c.Status(fiber.StatusBadRequest).JSON(fiber.Map{"message": "Invalid input"})
+    }
+
+    userID := data["user_id"]
+    username := data["username"]
+
+    var user entity.Account
+    if err := database.DB.First(&user, userID).Error; err != nil {
+        return c.Status(fiber.StatusNotFound).JSON(fiber.Map{"message": "User not found"})
+    }
+
+    user.Username = username
```

Comment:
nit: you could directly assign here the username, and remove line 22. You can do that here because the function here is pretty simple and easy to read/understand.

Example:
```go
user.Username = data["username"]
```